addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1599,6 +1599,21 @@ options.registry.anchor = options.Class.extend({
     },
 });
 
+/**
+ * Controls box properties.
+ */
+options.registry.Box = options.Class.extend({
+    /**
+     * @override
+     */
+    selectClass(previewMode, widgetValue, params) {
+        this._super(...arguments);
+        if (!widgetValue && params.activeValue === 'shadow') {
```

Comment:
More generic with: `params.possibleValues.includes('shadow')` (so that this still works if we someday add the shadow-sm/shadow-lg classes as an option).